agent/config/.env.example

```
@@ -0,0 +1,44 @@
+#--- APP ---#
+ENVIRONMENT="local"
+CORS_ORIGINS=["http://localhost:8000", "https://localhost:8000", "http://localhost", "https://localhost"]
+SERVER_HOST="https://new_project_name.dev"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)